packages/node_modules/@webex/internal-plugin-mercury/README.md

```diff
@@ -39,18 +41,25 @@ This can be done by configuring an agent as part of a DefaultMercuryOptions conf
 
 ```js
 const webex = require(`webex`);
+const url = require('url');
 const HttpsProxyAgent = require('https-proxy-agent');
 
-let httpsProxyAgent = new HttpsProxyAgent(url.parse(proxyUrl));
+// init webex
+let config = {
+  credentials: process.env.TOKEN
+  //...and anything else you need
+};
 
-webex.init({
-	config: {
-	  defaultMercuryOptions: {
-		agent: httpsProxyAgent
-	  },
-	 ...
-	}
-});
+let proxyUrl = process.env.HTTP_PROXY || null;
+
+if (proxyUrl) {
+  let httpsProxyAgent = new HttpsProxyAgent(url.parse(proxyUrl));
+  config.defaultMercuryOptions = {
+    agent: httpsProxyAgent
+  };
+}
+
+webex.init(config);
```

Comment:
Have you verified that this works? I think the shape of the object to provide an access token to `init` should be:

```js
webex.init({
  credentials: process.env.TOKEN
  config: {
    defaultMercuryOptions: {
      agent: httpsProxyAgent
    }
  }
});
```

Reply:
Confirmed, you need the credentials token at the same level as "config".

Reply:
OK